src/boot/store.js

```diff
@@ -27,4 +38,6 @@ export const restore = (onFinished?: () => void) =>
     onFinished,
   );
 
-export default store;
+const exportStore = store; // Just a hack to avoid eslint error of exporting non const value.
```

Comment:
This was not required but Eslint was too fixated that let declared variables must not be imported. So its just a hack to satisfy eslint and is not at all required.